app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1><%= @mechanic.name %></h1>
+<h1><%= @mechanic.years_experience %></h1>
+<% @mechanic.rides.each do |ride| %>
+    <h3><%= ride.name %></h3>
+<% end %>
+
+<%= form_with url: "/ride_mechanics", method: :post, data: { turbo: false } do |form| %>
```

Comment:
Great work! Another option for this path could be: "/mechanics/:mechanic_id/rides" which is also RESTful, and in this case the mechanic_id could go in the path if you didn't want to have to make a hidden field (but your implementation is totally fine, this is just another alternative)